misc/userscripts/password_fill

```diff
@@ -298,16 +374,19 @@ secret_backend() {
 }
 # =======================================================
 
-# load some sane default backend
-reset_backend
-pass_backend
 # load configuration
 QUTE_CONFIG_DIR=${QUTE_CONFIG_DIR:-${XDG_CONFIG_HOME:-$HOME/.config}/qutebrowser/}
 PWFILL_CONFIG=${PWFILL_CONFIG:-${QUTE_CONFIG_DIR}/password_fill_rc}
 if [ -f "$PWFILL_CONFIG" ] ; then
     # shellcheck source=/dev/null
     source "$PWFILL_CONFIG"
 fi
+BACKEND_CLASS="$backend"_backend
+
+# load some sane default backend
+reset_backend
+$BACKEND_CLASS
+
 init
```

Comment:
This changes the way `password_fill` is configured; but I don't see any reason to change it. I strongly recommend that the lines above containing `reset_backend` and `pass_backend` are kept and that the other lines are not inserted.

If you want to use the `last_pass_backend`, then just put `last_pass_backend` in your `password_fill_rc`.